cloud_governance/common/clouds/aws/savingsplan/savings_plans_operations.py

```diff
@@ -79,3 +83,50 @@ def get_savings_plans_list(self, states: list = [], **kwargs):
             response = self.savings_plan_client.describe_savings_plans(**kwargs)
             results[self.SAVINGS_PLANS].append(response.get(self.SAVINGS_PLANS))
         return results[self.SAVINGS_PLANS]
+
+    def get_monthly_active_savings_plan_summary(self):
```

Comment:
Where you calc the left day of the year and calc resource cost for it ?

Reply:
No, this is for savings plan.